### DISCUSSION TITLE
Error searching for guerrilla mail

### DISCUSSION THREAD

**Discussion by katastrophe92** [ImranR98/Obtainium#1274] [Q&A]
Hey there! I had guerrilla mail installed 3 weeks ago without a problem. Today I Uninstalled because every time obtanium searched for updates gave me this error. Then I tried to add and install again but still it gave me this error. Fdroid and gitlab sources showed the same error. 

![SmartSelect_20240111_094551.jpg](https://github.com/ImranR98/Obtainium/assets/103950507/cdb71765-e72d-4692-b329-992d37047a2e)

**Comment by DarkCrypt** [comment]
That repo is dead.
